### DISCUSSION TITLE
Better responsive images with srcset and sizes attributes | Leo's dev blog – stories, insights, and ideas

### DISCUSSION THREAD

**Discussion by giscus** [hta218/leohuynh.dev#71] [Announcements]
# Better responsive images with srcset and sizes attributes | Leo's dev blog – stories, insights, and ideas

A guide to using the srcset and sizes attributes to create responsive images

https://www.leohuynh.dev/blog/better-responsive-image-with-srcset-and-sizes-attributes

**Comment by hta218** [comment]
#### Hey, thanks for checking out this post! 💡

If you found it useful (or have some ideas for improvement), I’d love to hear from you! Leave a comment below with your thoughts, or give a reaction to show your support. It means a lot to me!

Cheers 🍺
